.github/actions/test-base/action.yml

```diff
@@ -15,19 +15,36 @@ runs:
   steps:
     - name: repository checkout step
       uses: actions/checkout@v4
+
     - name: update tracking reference step
       run: git remote set-branches origin main
       shell: bash
+
     - name: shallow clone update step
       run: git fetch --depth 1
       shell: bash
+
     - name: python environment step
       uses: actions/setup-python@v4
       with:
         python-version: ${{ inputs.python-version-identifier }}
-    - name: dependencies installation step
+
+    - name: Display Python version
+      run: python -c "import sys; print(sys.version)"
+      shell: bash
```

Comment:
This action is called with a python version input, and it says so in the job name. Is there a situation where this will add extra information?

Reply:
* success control
* adds the information in another place, making it easiere to access there